tests/Extensions/DecimalExtensionsShould.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Kros.Extensions;
+using Xunit;
+
+namespace Kros.Utils.UnitTests.Extensions
+{
+    public class DecimalExtensionsShould
+    {
+        [Fact]
```

Comment:
I suggest using `[Theory]` instead of `[Fact]` and adding more test data.

Reply:
Resolved